hack/quick_install.sh

```diff
@@ -17,11 +17,11 @@ Help() {
    echo
 }
 
-export ISTIO_VERSION=1.20.4
-export KNATIVE_OPERATOR_VERSION=v1.14.5
-export KNATIVE_SERVING_VERSION=1.13.1
+export ISTIO_VERSION=1.23.2
+export KNATIVE_OPERATOR_VERSION=v1.15.7
```

Comment:
KNATIVE 1.16 is planned to release next week. why don't we wait 1 week and use that?